src/datafactory/azext_datafactory/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isExperimental": true,
-    "azext.minCliCoreVersion": "2.3.1"
+    "azext.minCliCoreVersion": "2.11.0"
```

Comment:
what's required in 2.11.0 to support the new extension command?

Reply:
there's a msrest upgrade that is needed. https://github.com/Azure/azure-cli-extensions/issues/2082

Reply:
And because we have bumpped to newer python sdk generator. they require a new azure-mgmt-core.

Reply:
thanks @qiaozha . Does this apply to all new codes generated with latest generator?

Reply:
yes